packages/editor/src/components/post-saved-state/style.scss

```diff
@@ -44,3 +43,10 @@
 		}
 	}
 }
+
+// Fix for draft button glitch on Small desktop (e.g. 1024px) windows os. while Top Toolbar is enabled.
+.editor-post-save-draft,
+.editor-post-switch-to-draft,
```

Comment:
Per coding guidelines, styles should not be assigned outside the components own folder:

>A component's class name should never be used outside its own folder (with rare exceptions such as _z-index.scss). 

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#css

Instead, these should be assigned in stylesheets specific to the folder of the component to which they apply.

`packages/editor/src/components/post-switch-to-draft-button/style.scss` (should be created and [imported from `packages/editor/style.scss`](https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/style.scss))

`.editor-post-save-draft` does not appear to adhere to conventions, and since this was already the case, it would be fine to remain here.